pages/common/gist.md

```diff
@@ -23,10 +23,10 @@
 
 `gist -l`
 
-- List all gists for the currently logged in user:
+- List all public gists for any user:
```

Comment:
The command to list gists for currently logged user is just above : 

> - List your public and private gists:
>
 > `gist -l`